swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md

```diff
@@ -481,7 +475,7 @@ Invoke the defined resource function by sending the `POST` request below to `htt
 
 ```
 curl --location --request POST 'http://localhost:8080/employees/' \
-    --header 'Content-Type: text/plain' \
+    --header 'Content-Type: application/json' \
     --data-raw '{
```

Comment:
curl -X POST http://localhost:8080/employees/ \
     -H 'Content-Type: application/json' \
     -d '{

Reply:
Fixed in [2d141dd](https://github.com/ballerina-platform/ballerina-dev-website/pull/5725/commits/2d141dd617110bf5c2a358ec9e1ae9d0fc966003)